_shows/17_18/dirk_gently.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Dirk Gently's Holistic Detective Agency
+playwright: Douglas Adams
+adaptor: "James Goss & Arvind Ethan David"
+period: Spring
+season: In House
+season_sort: 300
+date_start: 2018-05-09
+date_end: 2018-05-12
+venue: New Theatre
+
+cast:
+- role: Dirk Gently
+  name: Beth Mullen
+- role: Reg
```

Comment:
Full character name is `Regius Professor Chronotis`, dunno if we should use that instead?